server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -6748,6 +6737,31 @@ public Pair<List<? extends NetworkOffering>, Integer> searchForNetworkOfferings(
         }
     }
 
+    private boolean allowNetworkOfferingWithNullTag(Long zoneId, List<String> pNtwkTags) {
+        boolean allowNullTag = false;
+        final List<PhysicalNetworkVO> pNtwks = _physicalNetworkDao.listByZoneAndTrafficType(zoneId, TrafficType.Guest);
```

Comment:
```suggestion
    private boolean allowNetworkOfferingWithNullTag(Long zoneId, List<String> physicalNetworkTags) {
        boolean allowNullTag = false;
        final List<PhysicalNetworkVO> physicalNetworks = _physicalNetworkDao.listByZoneAndTrafficType(zoneId, TrafficType.Guest);
```
I think we should make variable names readable and understandable to avoid confusion.

Reply:
Done 👍